README.md

```diff
@@ -1,2 +1,72 @@
 # floripa-routes
 Hybrid react.js app for search routes in Florianópolis.
+
+## Requirements
+
+- [Node.js](https://nodejs.org/)
+- [Phonegap](http://phonegap.com)
+
+```
+npm install -g phonegap
+```
+
+### Optional
+
+- [Android SDK](http://developer.android.com/sdk/installing/index.html?pkg=tools)
+- [Cordova](https://cordova.apache.org/)
+
+## Setup
+
+```
+npm install
+```
+
+## Run the app
+
+You can run this app in the [browser](http://localhost:3030):
+
+```
+npm start
+```
+
+To run on mobile through phonegap:
+
+```
+npm run phonegap
```

Comment:
You're right! The input text value was not passed to the search function. Fixed.